var/spack/repos/builtin/packages/camp/package.py

```diff
@@ -0,0 +1,46 @@
+# Copyright 2013-2020 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class Camp(CMakePackage, CudaPackage):
+    """
+    Compiler agnostic metaprogramming library providing concepts,
+    type operations and tuples for C++ and cuda
+    """
+
+    homepage = "https://github.com/LLNL/camp"
+    git      = "https://github.com/LLNL/camp.git"
+
+    version('develop', branch='master', submodules='True')
+    version('master', branch='master', submodules='True')
```

Comment:
You should only need one of these.  I'd stick with `master` since that is what the branch is called.

Reply:
Done